caesars-palace/blocks/stats/stats.css

```diff
@@ -21,7 +21,9 @@ div.stats-container .stats > div > div {
 
 /* Stats Title */
 div.stats-container .stats h1 {
-    font-family: var(--montserrat-font-family);
+    --heading-font-family: 'Montserrat', sans-serif; /* Override var locally */
+
+    font-family: var(--heading-font-family);
```

Comment:
You shouldn't have to include this line. The updated var will automatically get applied to the base style that uses the same var name.

Reply:
i thought it was timing scope thing (styles define the set on h1 then my block render after....was wrong). got it. fixed it.